WordPressAuthenticator/Authenticator/WordPressAuthenticatorConfiguration.swift

```diff
@@ -88,5 +95,6 @@ public struct WordPressAuthenticatorConfiguration {
         self.showLoginOptions = showLoginOptions
         self.enableSignInWithApple = enableSignInWithApple
         self.enableUnifiedAuth = enableUnifiedAuth
+        self.enableUnifiedSiteAddress = enableUnifiedSiteAddress
```

Comment:
I don't know how/where you planned on checking both `enableUnifiedAuth` and `enableUnifiedSiteAddress`, but what do you think about making this:
`self.enableUnifiedSiteAddress = enableUnifiedAuth && enableUnifiedSiteAddress`

So `enableUnifiedSiteAddress` in only `true` if both are `true`?

Reply:
Oh I like it! I'll change it to this.